app/src/containers/Dashboard/Dashboard.js

```diff
@@ -1,7 +1,52 @@
 import React from 'react';
+import { Link } from 'react-router';
+import { Nav, NavItem, Tab, Row, Col } from 'react-bootstrap';
+
+import EventsList from './../Events/EventsList';
 
 export default class Dashboard extends React.Component {
-  render () {
-    return <h1>Here will be dashboard</h1>;
+
+  render() {
+    return (
+      <div>
+        <h1>Dashboard</h1>
+        <Tab.Container id='left-tabs-example' defaultActiveKey='first'>
+          <Row className='clearfix'>
+            <Col sm={4}>
+              <Nav bsStyle='pills' stacked>
+                <NavItem eventKey='first'>
```

Comment:
You can do something like:
 `['My Cabinet', 'My events'...].map((title, index) => 
  <NavItem eventKey={index}>
                   {title}
 </NavItem>
)`